Utilities/Thread.h

```diff
@@ -36,6 +36,17 @@ enum class thread_state : u32
 	mask = 3
 };
 
+static inline thread_class get_thread_class(u32 thread_id_type)
+{
+	switch (thread_id_type)
+	{
+	case 1: return thread_class::ppu;
```

Comment:
This function can replaced with a simple cast.
thread_class::ppu = 1, thread_class::spu = 2, thread_class::rsx = 0x55

I dont remember where but its used in performance-sensituve context.